README.md

```diff
@@ -52,14 +57,6 @@ Run Guillotina (it requires Docker):
 npm run guillotina
 ```
 
-Create a Guillotina container for our project:
```

Comment:
I don't think Guillotina does that automatically. I assume you already had a site created in your Postgres docker image.